blog.php

```diff
@@ -1,9 +1,20 @@
 <?php include 'includes/header.php';?>
 <div class="card-component-container bottom-spacer">
-	<div class="row-container row-container--horizontal">
-	</div> <!-- End of row-container -->
+    <div class="row-container row-container--horizontal">
+    </div> <!-- End of row-container -->
 </div> <!-- End of card-component-container -->
+<div class="blog-sidebar-component">
```

Comment:
Change this class name to be just `sidebar-container` also change the class name in the sass file.

Reply:
done.